<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServerForm</class>
 <widget class="QWidget" name="TcpServerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="listenPortLabel">
     <property name="text">
      <string>端口：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="listenPortLE"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="listenPortBT">
     <property name="text">
      <string>监听端口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListWidget" name="portListWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
